pkg/serverless/trace/inferredspan/inferred_span.go

```diff
@@ -0,0 +1,66 @@
+package inferredspan
+
+import (
+	"strings"
+
+	"github.com/DataDog/datadog-agent/pkg/config"
+	"github.com/DataDog/datadog-agent/pkg/serverless/tags"
+	"github.com/DataDog/datadog-agent/pkg/trace/pb"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+const (
+	// tagInferredSpanTagSource is the key to the meta tag
+	// that lets us know whether this span should inherit its tags.
+	// Expected options are "lambda" and "self"
+	tagInferredSpanTagSource = "_inferred_span.tag_source"
+
+	// additional function keys to ignore
+	functionVersionTagKey = "function_version"
+	coldStartTagKey       = "cold_start"
+)
+
+var functionTagsToIgnore = [...]string{
```

Comment:
Is it possible to create an allow list instead?
I have the feeling that we will forget to add any new tags to this list and cause a breaking change by accident

Reply:
No, unfortunately we don't know the full list of tags we should allow since the customer can potentially add custom tags. Forgetting to add any new tags to this list won't be a breaking change, it'll simply cause some unwanted tags to show up for managed service spans.. unless you meant breaking in another way.